### PR TITLE
Add description with links for the Product settings

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds a description with links for the Product settings.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/catalog/settings.html

### CODE REVIEW

File: src/catalog/settings.md [hunk 1]
```diff
@@ -9,3 +9,17 @@ The most frequently used product settings and attributes are located at the top
...
+|[Search Engine Optimization]({% link catalog/product-search-engine-optimization.md %})|Specifies the URL Key and metadata fields that are used by search engines to index the product.|
+|[Related Products, Up-Sells, and Cross-Sells]({% link catalog/related-products-up-sells-cross-sells.md %})|Is used to set up simple promotional blocks that present a selection of additional products that might be of interest to the customer.|
+|[Customizable Options]({% link catalog/settings-advanced-custom-options.md %})|Adds customizable options to a product.|
+|[Product in Websites]({% link catalog/settings-basic-websites.md %})| Identifies each website where the product is available, according to the store hierarchy.|
+|[Design]({% link catalog/settings-advanced-design.md %})|Allows a different theme to be applied to the product page, changes the column layout, determines where product options appear and enter custom XML code.|
+|[Gift options]({% link catalog/product-gift-options.md %})|Can be set at the product level to allow a gift message to be included during checkout.|
```
Comment (leslietilling) [inline src/catalog/settings.md:24]:
```suggestion
|[Gift options]({% link catalog/product-gift-options.md %})|Used to enable or disable a gift message option during checkout at the product level.|
```

File: src/catalog/settings.md [hunk 2]
```diff
@@ -9,3 +9,17 @@ The most frequently used product settings and attributes are located at the top
...
+|[Downloadable Information]({% link catalog/product-downloadable-information.md %})|Is used to generate the links to downloadable products and samples.|
```
Comment (leslietilling) [inline src/catalog/settings.md:25]:
```suggestion
|[Downloadable Information]({% link catalog/product-downloadable-information.md %})|Used to generate the links to downloadable products and samples.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Editorial fixes

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling, thank you for the review :) 
I have implemented your suggestions, could you please approve?

File: src/catalog/settings.md [hunk 3]
```diff
@@ -9,3 +9,17 @@ The most frequently used product settings and attributes are located at the top
...
+
+|Setting|Description|
+|--- |--- |
+|[Sources]({% link catalog/sources.md %})|The list of sources from which the product can be distributed.|
+|[Content]({% link catalog/product-content.md %})|Is used to enter and edit the main product description that appears on the product page.|
+|[Configurations]({% link catalog/product-configurations.md %})| The lists of any existing variations of the product and can be used to generate variations for use with the Configurable product type.|
+|[Product Reviews]({% link catalog/settings-advanced-product-reviews.md %})|The lists of all reviews that customers have submitted about the product.|
+|[Search Engine Optimization]({% link catalog/product-search-engine-optimization.md %})|Specifies the URL Key and metadata fields that are used by search engines to index the product.|
+|[Related Products, Up-Sells, and Cross-Sells]({% link catalog/related-products-up-sells-cross-sells.md %})|Is used to set up simple promotional blocks that present a selection of additional products that might be of interest to the customer.|
+|[Customizable Options]({% link catalog/settings-advanced-custom-options.md %})|Adds customizable options to a product.|
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/catalog/settings.md:21]: @leslietilling Could you please clarify what I need to change here? Seems your suggestion is the same with my

[the remark this answers is not in the document]
Comment (leslietilling) [reply src/catalog/settings.md:21]: I think this one was a GitHub UI glitch.  Ignore.

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request closed without merging**